src/lib/viewers/archive/Breadcrumbs.js

```diff
@@ -19,14 +19,21 @@ class Breadcrumbs extends React.PureComponent {
      * @return {Array<Object>} path items including name and path string
      */
     getPathItems = fullPath => {
+        if (fullPath === ROOT_FOLDER) {
+            return [{ name: __('root_folder'), path: ROOT_FOLDER }];
+        }
+
```

Comment:
Is there a new test the encompasses this behavior?

Reply:
> Is there a new test the encompasses this behavior?

Yeah, I modified the old test to encompass this behavior.

Reply:
> Is there a new test the encompasses this behavior?

done